weasyprint/css/computed_values.py

```diff
@@ -195,6 +195,37 @@ def compute(element, pseudo_type, specified, computed, parent_style,
     """
     from .validation.properties import PROPERTIES
 
+    def resolve_var(value, name, computed):
```

Comment:
Would it be possible to put the function outside of the `compute` function? Redefining the function each time `compute` is called takes useless time.